composer.json

```
@@ -0,0 +1,43 @@
+{
+    "name": "spryker-eco/unzer",
+    "type": "library",
+    "description": "Unzer PSP module",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.3",
+        "spryker/kernel": "^3.30.0",
+        "spryker/store": "^1.0.0",
+        "spryker/util-text": "^1.0.0",
+        "spryker/zed-request": "^3.0.0"
+    },
+    "require-dev": {
+        "phpstan/phpstan": "*",
+        "spryker/code-sniffer": "*",
+        "spryker/testify": "*",
+        "codeception/module-asserts": "^1.3"
+    },
+    "autoload": {
+        "psr-4": {
+            "SprykerEco\\": "src/SprykerEco/"
+        }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "scripts": {
+        "test": "vendor/bin/codecept run",
+        "cs-check": "phpcs -p -s --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml src/",
+        "cs-fix": "phpcbf -p --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml src/",
+        "stan": "phpstan analyse -c phpstan.neon -l 8 src/",
```

```suggestion
        "stan": "phpstan analyse -c phpstan.neon src/",
```

Please see comment at phpstan.neon for more details ...